prow.go

```diff
@@ -159,10 +159,11 @@ func (m *jobManager) stopJob(name, cluster string) error {
 		return err
 	}
 
-	// Deleting the prowjob pod will gracefully terminate template or step based jobs. In the case of
-	// steps, this includes running post steps.
-	klog.Infof("ProwJob pod for job %q will be deleted", name)
-	return m.clusterClients[cluster].CoreClient.CoreV1().Pods(m.prowNamespace).Delete(context.TODO(), name, metav1.DeleteOptions{})
+	klog.Infof("ProwJob pod for job %q will be aborted", name)
+	pj.SetComplete()
```

Comment:
This is wrong, it will result in us not terminating the pod as we ignore Completed jobs:
https://github.com/kubernetes/test-infra/blob/01baa4728fcb61d887c858bf4af0f7ca094fd406/prow/plank/reconciler.go#L735-L738

Reply:
Removed